tests/BaseTestCase.php

```diff
@@ -166,6 +166,8 @@ protected function getGitConfigHelper()
     {
         $helper = $this->prophesize('Gush\Helper\GitConfigHelper');
         $helper->setHelperSet(Argument::any())->willReturn();
+        $helper->ensureRemoteExists(Argument::cetera())->will(function ($args) { return implode('_', $args); });
```

Comment:
It's the only way we can mock things as PHPUnit mocks are only expectations and there is no way to say: return `...` when the parameters are `...`.